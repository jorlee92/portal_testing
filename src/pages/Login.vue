<template>
  <div class="content">
    <p>
    this be a login page
    </p>
    <LoginForm/>
  </div>
</template>

<script>
import LoginForm from '@/components/LoginForm'
export default {
  components: {
    LoginForm,
  },
}
</script>

<style lang="sass" scoped>
</style>