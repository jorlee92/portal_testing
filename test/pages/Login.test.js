import { shallow } from 'vue-test-utils'
import Login from '@/pages/Login.vue'

describe('Login', () => {
  let wrapper
  beforeEach(() => {
    wrapper = shallow(Login)
  })
  it('should render the view correctly', () => {
    expect(wrapper.element).toMatchSnapshot()
  })
})
