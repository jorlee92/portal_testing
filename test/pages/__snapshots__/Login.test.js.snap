// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login should render the view correctly 1`] = `
<div
  class="content"
>
  <p>
    
  this be a login page
  
  </p>
   
  <!---->
</div>
`;
